src/Classes/MyRadio/MyRadioFormField.php

```diff
@@ -784,7 +784,7 @@ private function convertTime($timeString)
          * YES, PHP IS RETARDED.
          */
         $timeString = str_replace('/', '-', $timeString);
-        $time = (int) strtotime($timeString) - date('Z');
+        $time = (int) strtotime($timeString);
```

Comment:
Does this mean the displayed requested time (for the person scheduling) will always be UTC then?